### ISSUE TITLE
The votes and comments column next to each other are confusing

### ISSUE THREAD

**Issue by itislu** [42-student-council/website#167]
If you don't see the header, it's just two numbers next to each other.
Especially on mobile when I swipe to the right of the table to see the number of votes, I always get confused.

A possible solution could be:
- Add a small heart next to the number of votes and a small comment bubble symbol to the number of comments.

**Comment by leeoocca** [comment]
I agree! In general, as I was telling @leske42, it might be nicer to move to a card-based design, especially for mobile.

This way you could arrange the content more freely and with more icons without it becoming too filled:

```
+----------------------------+
| Title of the issue         |
+----------------------------+
| Yesterday at 15:42         |
+-------------+--------------+
| X likes     | X comments   |
+-------------+--------------+
```
